src/main/java/net/smartcosmos/dao/relationships/impl/RelationshipPersistenceService.java

```diff
@@ -273,11 +327,55 @@ private RelationshipEntity persist(RelationshipEntity relationshipEntity) throws
         }
     }
 
-    private List<RelationshipResponse> convert(List<RelationshipEntity> entityList) {
+    private List<RelationshipResponse> convertList(List<RelationshipEntity> entityList) {
         return entityList.stream()
             .map(o -> conversionService.convert(o, RelationshipResponse.class))
             .collect(Collectors.toList());
     }
 
-    // endregion
+    private Pageable buildPageable(Integer page, Integer size, SortOrder sortOrder, String sortBy) {
+
+        Sort.Direction direction = Sort.DEFAULT_DIRECTION; // TODO default value to service config
+        if (sortOrder != null) {
+            direction = RelationshipPersistenceUtil.getSortDirection(sortOrder);
+        }
+        if (sortBy == null) {
+            sortBy = RelationshipPersistenceUtil.getSortByFieldName("created"); // TODO default value to service config
+        }
+        if (page == null) {
+            page = 0; // TODO default value to service config
+        }
+        if (size == null) {
+            size = 20; // TODO default value to service config
+        }
+        return new PageRequest(page, size, direction, sortBy);
+    }
+
+
+    private boolean alreadyExists(String accountUrn, RelationshipCreate createRelationship) {
+
+        Optional<RelationshipResponse> existing = findSpecific(
+            accountUrn,
+            createRelationship.getSource().getType(),
+            createRelationship.getSource().getUrn(),
+            createRelationship.getTarget().getType(),
+            createRelationship.getTarget().getUrn(),
+            createRelationship.getRelationshipType());
+
+        if (existing.isPresent()) {
+            return true;
+        }
+        return false;
+    }
+
+    private void exceptionLogger(String methodName, Exception e, Object... args) {
```

Comment:
I don't think this method is an improvement. All the relevant information is already provided to call this method why bother hiding logic in another method.  Sometimes reuse is just burying what can and should be visible.